### ISSUE TITLE
The DP command implementation is duplicated (Bugzilla Bug 721)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/edk2#7133]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 721
Date: 2017-09-26T22:08:14+00:00
From: @jyao1
To:   @niruiyu
CC:   @niruiyu, yonghong.zhu

Duplicates:   #302
Last updated: 2018-01-17T05:14:04+00:00

**Comment by tianocore-issues** [comment]
## Comment 2791
Date: 2017-09-26 22:08:14 +0000
From: @jyao1

- Industry Specification: ---
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): @niruiyu

We have DP implementation on both ShellPkg\Library\UefiDpLib and PerformancePkg\Dp_App.

It is bad to duplicate the code.

We should only maintain one copy of the code.

**Comment by tianocore-issues** [comment]
## Comment 2798
Date: 2017-09-28 20:11:20 +0000
From: Yonghong Zhu &lt;<yonghong.zhu>&gt;

bug scrub: assign to Jaben to investigate and discuss in community first.

**Comment by tianocore-issues** [comment]
## Comment 3434
Date: 2018-01-17 05:11:23 +0000
From: @niruiyu

take it.

**Comment by tianocore-issues** [comment]
## Comment 3435
Date: 2018-01-17 05:14:04 +0000
From: @niruiyu

After #302 was resolved, duplicated DP implementation was removed as well.

*** This bug has been marked as a duplicate of bug 302 ***
